### PR TITLE
ACMS-4061: Fixed all failing tests

### PR DESCRIPTION
**Pull request by vishalkhode1**

### CODE REVIEW

**Review by apathak18 — APPROVED** [review]

**Review by deepakmishra2 — APPROVED** [review]

File: modules/acquia_cms_site_studio/acquia_cms_site_studio.module [hunk 1]
```diff
@@ -280,9 +280,18 @@ function acquia_cms_site_studio_modules_installed($modules, $is_syncing) {
...
+    // The module uses different paths in different releases. Hence, removing
+    // all possible paths to support all releases of collapsiblock module.
+    $paths = [
+      $module_path . '/theme/dist/css/collapsiblock.min.css',
+      $module_path . '/theme/dist/collapsiblock.css',
+      $module_path . '/theme/dist/style.css',
+    ];
+    foreach ($paths as $path) {
+      unset($css[$path]);
+    }
```
Comment (chandan-singh7929) [inline modules/acquia_cms_site_studio/acquia_cms_site_studio.module:295]: Can we use hook_library_info_alter, instead ?. Also I don't see file name `/theme/dist/style.css`

**Review by rajeshreeputra — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]
Overall changes looks good to me, though I have provided suggestion. Pease check if we can unset the library itself instead of removing files associated with library.

File: modules/acquia_cms_site_studio/acquia_cms_site_studio.info.yml [hunk 2]
```diff
@@ -5,11 +5,11 @@ core_version_requirement: ^9.5 || ^10
 type: module
 dependencies:
   - acquia_cms_common:acquia_cms_common
-  - cohesion_base_styles
-  - cohesion_custom_styles
-  - cohesion_elements
-  - cohesion_style_helpers
-  - cohesion_sync
-  - cohesion_website_settings
+  - cohesion_base_styles # phpcs:ignore
```
Comment (apathak18) [inline modules/acquia_cms_site_studio/acquia_cms_site_studio.info.yml:8]: Use at the top of the file
# phpcs:ignoreFile

Comment (vishalkhode1) [reply modules/acquia_cms_site_studio/acquia_cms_site_studio.info.yml:8]: That will ignore **ALL** PHPCS Rules across the file, but we want to ignore only the specific lines. That's why we have not added at the top.

**Pull request merged**